analytics/mapreduce/src/main/java/mil/nga/giat/geowave/analytic/mapreduce/kde/KDEJobRunner.java

```diff
@@ -431,4 +444,19 @@ protected int getCQLFilterArg() {
 		return 15;
 	}
 
+	protected void addJobClasspathDependencies(
+			final Job job,
+			final Configuration conf )
+			throws IOException,
+			URISyntaxException {
+		final String[] jars = conf.getTrimmedStrings(GEOWAVE_CLASSPATH_JARS);
```

Comment:
where does the value for this 'geowave.classpath.jars' get in the job configuration in the first place?